version/helper.go

```diff
@@ -31,6 +31,17 @@ var (
 
 const PackageVersionFileName = "package.version"
 
+var versionInitError error
+
+func init() {
+	versionInitError = InitVersionInformation()
```

Comment:
This is already called in another init that I commented on.

You should make the internal of this function `InitVersionInformation` use a `sync.Once` and store the error inside of the `sync.once` instead of on the outside. That will sure that this is initialized only once and the error is always set correctly.

Reply:
The init() function is guaranteed to be initialized only once already by the go runtime, so no extra synchronization mechanisms are necessary. The only concurrent access we might have is reading versionInitError after initialization but since the value of the variable never changes after initialization it is not a problem.